### PR TITLE
Add help block to input field

### PR DESCRIPTION
**Pull request by mlunoe**
This adds the ability to have help blocks in addition to error messages

### CODE REVIEW

File: src/Form/FieldInput.js [hunk 1]
```diff
@@ -222,6 +233,8 @@ FieldInput.propTypes = {
   handleEvent: React.PropTypes.func,
   // Optional label to add
   label: React.PropTypes.string,
+  // Optional hepl block
+  helpBlock: React.PropTypes.node,
```
Comment (jfurrow) [inline src/Form/FieldInput.js]: Let's add `helpBlockClass` to `propTypes`

Comment (mlunoe) [inline src/Form/FieldInput.js]: It's already there

Comment (jfurrow) [inline src/Form/FieldInput.js]: Ahh cool, I didn't realize that. Should we call `classnames` on it as we do in [here](https://github.com/mesosphere/reactjs-components/blob/dfea31110f87fe945419749fefde3fa84e2f08c7/src/Form/FieldInput.js#L94)?

Comment (mlunoe) [inline src/Form/FieldInput.js]: Arh, yes we should. Done

**Comment by mlunoe** [comment]
ping @jfurrow @philipnrmn

**Comment by jfurrow** [comment]
:ship:

File: src/Form/FieldInput.js [hunk 2]
```diff
@@ -222,6 +237,8 @@ FieldInput.propTypes = {
   handleEvent: React.PropTypes.func,
   // Optional label to add
   label: React.PropTypes.string,
+  // Optional hepl block
```
Comment (philipnrmn) [inline src/Form/FieldInput.js]: s/hepl/help

File: docs/src/Form/index.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ class FormExample extends React.Component {
       {
         fieldType: 'password',
         name: 'Password',
+        helpBlock: 'If you want to make your password strong you could use a password manager',
```
Comment (philipnrmn) [inline docs/src/Form/index.js]: Not a fan of this wording - the tone is at odds with what we're using elsewhere.

**Comment by philipnrmn** [comment]
No issue with the code, but I'm not sure I like the actual warning much! Happy to merge if we commit to reviewing it soon though.

File: src/Form/FieldInput.js (hunk 2 above)
Comment (mlunoe) [inline src/Form/FieldInput.js]: Done

File: docs/src/Form/index.js (hunk 3 above)
Comment (mlunoe) [inline docs/src/Form/index.js]: Changed it up a bit. Had a conversation offline and came up with something that we could both agree on

**Comment by mlunoe** [comment]
Fixed

**Pull request merged**